.github/workflows/ci-build.yaml

```
@@ -401,6 +401,14 @@ jobs:
       - run: yarn --cwd ui test
       - run: yarn --cwd ui lint
       - run: yarn --cwd ui deduplicate
+      # Depends deduplicate changes for dependabot-reviewer to work
```

```suggestion
      # Deduplicate UI deps for dependabot PRs
```

This isn't needed for `dependabot-reviewer`, just for dependabot. That Workflow will pass, but it won't merge because this step of the build fails